DS/DSassignment1/ArrayList1.java

```diff
@@ -0,0 +1,203 @@
+import java.util.Arrays;
+
+public class ArrayList1 {
+	
+	public static int defaultSize = 10;
+	public static int size=0;
+	public static Object data[] = new Object[defaultSize];
```

Comment:
Looking at the subsequent classes if think you have learnt 'Generics'. You should use that instead of 'Object'.